gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -130,6 +132,15 @@ export default function NxTabsPage() {
         {' '}<code className="nx-code">NxTile</code> where there is no tile header and the tabs are top most within
         the tile.
       </GalleryExampleTile>
+      <GalleryExampleTile title="NxTabs outside of NxTile Example"
+                          id="nx-tab-outside-tile-example"
+                          defaultCheckeredBackground={true}
+                          liveExample={NxTabsOutsideTileExample}
+                          codeExamples={NxTabsOutsideTileExampleCode}>
+        A basic example of how to use the <code className="nx-code">NxTabs</code> family of components in an
```

Comment:
The title says "Outside of NxTile" but the description says "in an NxTile". I think the former is correct, but that makes the whole description incorrect. Also shouldn't it be `nx-tile`?

Reply:
> The title says "Outside of NxTile" but the description says "in an NxTile". I think the former is correct, but that makes the whole description incorrect

Right the description was copypasta that I forgot to update. 9ff954cd

> Also shouldn't it be nx-tile?

Probably, but I was just going for consistency with what's already on this page.